IBPSA/Fluid/Geothermal/Borefields/BaseClasses/Boreholes/BaseClasses/Functions/internalResistancesTwoUTube.mo

```diff
@@ -1,5 +1,5 @@
 within IBPSA.Fluid.Geothermal.Borefields.BaseClasses.Boreholes.BaseClasses.Functions;
-impure function internalResistancesTwoUTube
+pure function internalResistancesTwoUTube
```

Comment:
Is the `pure` keyword required?

Reply:
It is not required as functions are `pure` by default. I removed the keyword.